src/ReverseProxy/Service/RuntimeModel/ClusterProxyHttpRequestOptions.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Net.Http;
+
+namespace Microsoft.ReverseProxy.RuntimeModel
+{
+    /// <summary>
+    /// Outgoing request configuration.
+    ///
+    /// Readonly copy of <see cref="Microsoft.ReverseProxy.Abstractions.ProxyHttpRequestOptions"/>.
+    /// </summary>
+    public readonly struct ClusterProxyHttpRequestOptions
+    {
+#if NET
+        public ClusterProxyHttpRequestOptions(TimeSpan? requestTimeout, Version version, HttpVersionPolicy? versionPolicy)
+        {
+            RequestTimeout = requestTimeout;
+            Version = version;
+            VersionPolicy = versionPolicy;
+        }
+#else
+        public ClusterProxyHttpRequestOptions(TimeSpan? requestTimeout, Version version)
+        {
+            RequestTimeout = requestTimeout;
+            Version = version;
+        }
+#endif
```

Comment:
This constructor should exist in both versions. We don't change/remove public API across versions if we can avoid it.
```suggestion
#endif
        public ClusterProxyHttpRequestOptions(TimeSpan? requestTimeout, Version version)
        {
            RequestTimeout = requestTimeout;
            Version = version;
        }

```

Reply:
Done.